lib/importproject.cpp

```diff
@@ -266,17 +266,27 @@ void ImportProject::fsParseCommand(FileSettings& fs, const std::string& command,
             pos++;
         if (pos >= command.size())
             break;
+        bool wholeArgQuoted = false;
+        if (command[pos] == '"') {
+            wholeArgQuoted = true;
+            pos++;
+            if (pos >= command.size())
+                break;
+        }
         if (command[pos] != '/' && command[pos] != '-')
             continue;
         pos++;
         if (pos >= command.size())
             break;
         const char F = command[pos++];
         if (std::strchr("DUI", F)) {
-            while (pos < command.size() && command[pos] == ' ')
+            while (pos < command.size() && command[pos] == ' ') {
                 ++pos;
+                wholeArgQuoted = false;
+            }
         }
-        std::string fval = readUntil(command, &pos, " =");
+        const char *readUntilChars = wholeArgQuoted ? " =\"" : " =";
```

Comment:
if wholeArgQuoted is true it shouldn't stop on space. Imagine:

     gcc "-IC:\Program Files\Foo" test.c

The include path should not be just `C:\Program`.